Startup.cs

```diff
@@ -27,8 +28,20 @@ public Startup(IConfiguration configuration, IWebHostEnvironment environment)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            // If configuration specifies mocking should be enabled, don't create HTTP clients and simply inject
+            // mocked services as singletons.
+            if (Configuration.GetValue<bool>("MockRequests"))
+                services.AddSingleton<IMetAPIService, MetAPIServiceMocked>();
+            else
+                services.AddHttpClient<IMetAPIService, MetAPIService>(client =>
+                {
+                    client.DefaultRequestHeaders.Accept.Add(
+                        new MediaTypeWithQualityHeaderValue("application/json"));
+                    client.DefaultRequestHeaders.UserAgent.ParseAdd(
+                        "DVT/1.0 (fredrik.malmo@icloud.com)"); // api.met.no requires contact email
+                });
+
```

Comment:
When we extend this to have, say, 10 different API's, do we add each of them here?

Reply:
Then we would have to add one line for each. The benefit is that you can configure each HTTP client separately. The downside is that you have to write 10 lines which may look a bit inefficient, but I believe this is a sensible solution. Any alternatives you can think of?

Reply:
Not at the moment, it's probably the best approach we have for now. Would be cool to find a way to make it look less bloated _if_ we at some point have 10+ API's in the future. Then again, this might be the right way to do it, so I have no feelings as to how it _should_ be.